<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QFrame {
	border: 2px solid black;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QFrame" name="leftMenu">
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>0</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Season</string>
       </property>
      </widget>
      <widget class="QDateEdit" name="dateEdit">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>110</width>
         <height>22</height>
        </rect>
       </property>
       <property name="maximumDateTime">
        <datetime>
         <hour>23</hour>
         <minute>59</minute>
         <second>59</second>
         <year>2024</year>
         <month>12</month>
         <day>31</day>
        </datetime>
       </property>
       <property name="maximumDate">
        <date>
         <year>2024</year>
         <month>12</month>
         <day>31</day>
        </date>
       </property>
       <property name="minimumDate">
        <date>
         <year>2023</year>
         <month>1</month>
         <day>1</day>
        </date>
       </property>
       <property name="displayFormat">
        <string>yyyy</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboDriver">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>121</width>
         <height>32</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="comboTeams">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>121</width>
         <height>32</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Driver</string>
       </attribute>
       <widget class="QLabel" name="driverPhoto">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>231</width>
          <height>191</height>
         </rect>
        </property>
        <property name="text">
         <string>Driver</string>
        </property>
       </widget>
       <widget class="QLabel" name="countryPhoto">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>20</y>
          <width>311</width>
          <height>191</height>
         </rect>
        </property>
        <property name="text">
         <string>Country</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Team</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QFrame" name="frame_3">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>50</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <widget class="QLabel" name="lblTeamName">
           <property name="geometry">
            <rect>
             <x>70</x>
             <y>0</y>
             <width>111</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
          <widget class="QLabel" name="lblTeamNamePre">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>0</y>
             <width>51</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Team:</string>
           </property>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="FrameTeamConfig">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <layout class="QVBoxLayout" name="LayoutTeamConfig"/>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Add Races</string>
       </attribute>
       <widget class="QFrame" name="frame">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>241</width>
          <height>221</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="3" column="0">
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>Date:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Name:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="QSpinBox" name="spbRaceRound">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>24</number>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_2">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Track:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>Round:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QLineEdit" name="leRaceName"/>
         </item>
         <item row="0" column="3">
          <widget class="QComboBox" name="comboTrackNames">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="3" column="3">
          <widget class="QDateEdit" name="deRaceDate">
           <property name="dateTime">
            <datetime>
             <hour>20</hour>
             <minute>0</minute>
             <second>0</second>
             <year>2024</year>
             <month>1</month>
             <day>1</day>
            </datetime>
           </property>
           <property name="maximumDateTime">
            <datetime>
             <hour>19</hour>
             <minute>59</minute>
             <second>59</second>
             <year>2025</year>
             <month>12</month>
             <day>31</day>
            </datetime>
           </property>
           <property name="minimumDateTime">
            <datetime>
             <hour>20</hour>
             <minute>0</minute>
             <second>0</second>
             <year>2022</year>
             <month>12</month>
             <day>31</day>
            </datetime>
           </property>
          </widget>
         </item>
         <item row="4" column="3">
          <widget class="QPushButton" name="btnAddRace">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Add Race</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QFrame" name="frame_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>370</y>
          <width>581</width>
          <height>131</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QComboBox" name="comboBox_3">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>60</y>
           <width>171</width>
           <height>32</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>10</y>
           <width>81</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>AddResult</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>40</y>
           <width>81</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Race</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_6">
         <property name="geometry">
          <rect>
           <x>190</x>
           <y>40</y>
           <width>81</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Coureur</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_7">
         <property name="geometry">
          <rect>
           <x>300</x>
           <y>40</y>
           <width>81</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Positie</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_8">
         <property name="geometry">
          <rect>
           <x>390</x>
           <y>40</y>
           <width>81</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Pole</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_9">
         <property name="geometry">
          <rect>
           <x>480</x>
           <y>40</y>
           <width>81</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Punten</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="checkBox">
         <property name="geometry">
          <rect>
           <x>390</x>
           <y>60</y>
           <width>85</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Pole</string>
         </property>
        </widget>
        <widget class="QSpinBox" name="spinBox">
         <property name="geometry">
          <rect>
           <x>310</x>
           <y>60</y>
           <width>42</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QSpinBox" name="spinBox_2">
         <property name="geometry">
          <rect>
           <x>480</x>
           <y>60</y>
           <width>42</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QComboBox" name="comboBox_4">
         <property name="geometry">
          <rect>
           <x>180</x>
           <y>60</y>
           <width>121</width>
           <height>32</height>
          </rect>
         </property>
        </widget>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>37</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
